Helper/Data.php

```diff
@@ -418,6 +424,17 @@ public function isSendCustomerDataToNostoEnabled(StoreInterface $store = null)
         return (bool)$this->getStoreConfig(self::XML_PATH_SEND_CUSTOMER_DATA, $store);
     }
 
+    /**
+     * Returns if category sorting is enabled
+     *
+     * @param StoreInterface|null $store the store model or null.
+     * @return bool the configuration value
+     */
+    public function isCategorySortingEnabled(StoreInterface $store = null)
```

Comment:
Why can this be null?

Reply:
It is the standard that we are using for each config related method.